guide/C/ch_invest.xml

```diff
@@ -1530,18 +1541,18 @@ Income
         <guibutton>Options</guibutton> icon in the <emphasis>Menubar</emphasis> or selecting
         <menuchoice>
           <guimenu>Edit</guimenu><guimenuitem>Report Options</guimenuitem>
-        </menuchoice>
-        . <guilabel>Price Source</guilabel> determines how accounts denominated in commodities
+        </menuchoice>.
+        <guilabel>Price Source</guilabel> determines how accounts denominated in commodities
         different from the report currency are converted to the report currency. Depending on the
         report the selector may appear in either the <guilabel>General</guilabel>, the
         <guilabel>Commodities</guilabel>, or the <guilabel>Display</guilabel> tab of the
         <guilabel>Report Options</guilabel> dialog box.
-        <note>
+         <note>
           <para>In the example below, the report is a customization of the <emphasis role="strong">Average
```

Comment:
`<emphasis role="strong">` can we replace this <q>HTML style</q> by better markups?

Reply:
The figure/image id="invest-stockvalue-report-options" seems to be Reports->Asset&Liabirities->Asset Chart,
not Examples->Avarage Balance.

Reply:
ab6c9581